unit_test/dependency_constraints_test.py

```diff
@@ -0,0 +1,22 @@
+from cibuildwheel.util import DependencyConstraints
+import os
+
+
+def test_defaults():
+    dependency_constraints = DependencyConstraints.with_defaults()
+
+    project_root = os.path.dirname(os.path.dirname(__file__))
+    resources_dir = os.path.join(project_root, 'cibuildwheel', 'resources')
+
+    assert os.path.samefile(
+        dependency_constraints.base_file_path,
+        os.path.join(resources_dir, 'constraints.txt')
+    )
```

Comment:
There are separated constraints files for python 3.5 and 3.6. Why it is not tested?

Reply:
good point, yes.